semanticdb/semanticdb3/semanticdb3.md

```diff
@@ -1293,35 +1293,41 @@ Notes:
   * `PROTECTED_THIS`: set for getters of `protected[this]` values.
   * `PROTECTED_WITHIN`: set for getters of `protected[...]` values.
 
-**Variable declarations and definitions** [\[41\]][41] are represented
-similarly to values (see above). Concretely, the following rules describe
-symbols created for variables:
-* Whenever a `VAL` symbol would be created for a value, a `VAR` symbol is
-  created for a variable having the same properties, signature and
-  accessibility.
-* Whenever a `GETTER` symbol would be created for a value, a `GETTER` symbol
-  is created for a variable having the same properties, signature and
-  accessibility.
-* Whenever a `GETTER` symbol would be created for a value, a `SETTER` symbol
-  is created for a variable, having the same properties (except `IMPLICIT`)
-  and accessibility
-  and (for `var x: T`) name `x_=`
-  and signature `MethodType(List(), List(List(<x$1>)), <Unit>)`
-  with the synthetic parameter `x$1` having signature `<T>`.
-* Whenever a `PARAMETER` symbol would be created for a value, a `PARAMETER`
-  symbol is created for a variable having the same properties, signature
-  and accessibility.
-* Variable declarations and definitions cannot be `lazy`, so variable symbols
-  cannot be `LAZY` either.
-* Only corresponding `GETTER` and `PARAM` symbols for `implicit` member
-  variables and only corresponding `VAR` symbols for `implicit` local
-  variables can be `IMPLICIT`. `SETTER` symbols can never be `IMPLICIT`.
+**Variable declarations and definitions** [\[41\]][41] are represented by
+multiple symbols, with the exact number of symbols, their kinds, properties,
+signatures and accessibilities dependent on the corresponding value:
+* Local symbol of kind `LOCAL` is created for all local variables.
+* Field symbol of kind `FIELD` is created for non-`ABSTRACT` member variables
+  to model the field generated by the Scala compiler.
+* Getter and setter symbols of kind `METHOD` are created for all
```

Comment:
Repeating question from elsewhere, is there a property to distinguish the getter from the setter?

Reply:
Updated the spec.